### ISSUE TITLE
Unexpected results when attributes are on new lines

### ISSUE THREAD

**Issue by MirukaJP** [markuplint/markuplint#285]
I think I found this problem.
The result of Prettier formatting the code is the following code.
```html
<ul class="example">
  <li class="example-item"><!-- The content of this element is invalid. -->
    <a
      id="example-id"
      class="example-class"
      href="#example">
      <span
        class="
          example__icon
          glyphicons
          glyphicons-menu-hamburger
        "></span>
      <span
         class="
           example__icon
           glyphicons
           glyphicons-chevron-left
         "></span>
      <span class="example__label"
         >Close</span
      >
    </a>
  </li>
</ul>
```

At this time, the content of `<li class="example-item">` is determined to be invalid.
However, after reformatting the code as follows, it was determined to be correct.

```html
<ul class="example">
  <li class="example-item">
    <a id="example-id" class="example-class" href="#example">
      <span class="example__icon glyphicons glyphicons-menu-hamburger"></span>
      <span class="example__icon glyphicons glyphicons-chevron-left"></span>
      <span class="example__label">Close</span>
    </a>
  </li>
</ul>
```

The content is the same, but the results are different.

**Comment by YusukeHirao** [comment]
@MirukaJP 
Thank you for your reporting.

Could you show me the error message?

I grasped an error arise from the [`indentation`](https://markuplint.dev/rules/indentation) rule on my environment.
```
$ markuplint test/fixture/010.html
<markuplint> warning: インデントはタブにしてください (indentation) /markuplint/test/fixture/010.html:3:1
  2: →   <li•class="example-item"><!--•The•content•of•this•element•is•invalid.•-->
  3:      <a
  4: →   →   id="example-id"
<markuplint> warning: インデントを深くしてください (indentation) /markuplint/test/fixture/010.html:7:1
  6: →   →   href="#example">
  7:            <span
  8: →   →   ••class="
<markuplint> warning: インデントを深くしてください (indentation) /markuplint/test/fixture/010.html:13:1
  12: →   →   ••"></span>
  13:           <span
  14: →   →   •••class="
<markuplint> warning: インデントを深くしてください (indentation) /markuplint/test/fixture/010.html:19:1
  18: →   →   •••"></span>
  19:           <span•class="example__label"
  20: →   →   •••>Close</span
<markuplint> warning: インデントはタブにしてください (indentation) /markuplint/test/fixture/010.html:22:1
  21: →   →   >
  22:     </a>
  23: →   </li>
<markuplint> warning: インデントはタブにしてください (indentation) ¥/markuplint/test/fixture/010.html:24:1
  23: →   </li>
  24:   </ul>
  25: 
error Command failed with exit code 1.
```

What the rule type do you found?

**Comment by MirukaJP** [comment]
In this case, the following error message is displayed.

> HTMLの仕様においてli要素の内容が不正です (permitted-contents)

From my research, this was confirmed in the following environment.

- Windows 10
- Visual Studio Code v1.62.3
- markuplint v1.11.4
- markuplint for VS Code v2.0.3
- Prettier v9.0.0
- Node.js v16.13.0
- npm v8.1.4

The Prettier setting is the default.

```
"printWidth": 80,
"tabWidth": 2,
"useTabs": false,
```

These are the items that may be relevant.


However, I did not get this error in the macOS environment.
Everything is the same except the OS.

- macOS Big Sur v11.2.3

**Comment by YusukeHirao** [comment]
@MirukaJP 

I found this cause.
It is that the code has the CRLF new line.

markuplint does not support the CRLF.
It is a known issue #31.

I'm sorry, I don't think I will resolve it yet.
